classes/Database.php

```diff
@@ -0,0 +1,186 @@
+     *     Stores the username.
+     *   @param string $password
+     *     Stores the password of the database.
+     *   @param string $dbName
+     *     Stores the name of the database.
+     * 
+     *   @return void
+     *     Only stores the object of the database.
+     */
+    public function __construct(string $serverName, string $userName, string $password, string $dbName) {  
+      $this->conn = $this->connect($serverName, $userName, $password, $dbName);
+    }
+
+    /**
+     * This function is used to connect to the database.
+     * 
+     *   @param string $serverName
+     *     Stores the servername.
+     *   @param string $userName
+     *     Stores the username.
+     *   @param string $password
+     *     Stores the password of the database.
+     *   @param string $dbName
+     *     Stores the name of the database.
+     * 
+     *   @return mixed
+     *     Returns mysqli object or error message while connecting to the
+     *     database.
+     */
+    public function connect($serverName, $userName, $password, $dbName) {
+      // Create connection.
+      $conn = new mysqli($serverName, $userName, $password, $dbName);
+      // Check connection.
+      if ($conn->connect_error) {
+        die("Connection failed: " . $conn->connect_error);
+      }
+      return $conn;
+    }
+
+    /**
+     * This function is used to fetch the details of the user from the database.
+     * 
+     *   @param string $email
+     *     Stores the Email-ID of the user.
+     * 
+     *   @return mixed
+     *     Returns an array containing the details of the user if user exists
+     *     otherwise NULL.
+     */
+    public function userDetails(string $userName) {
+      $data = "SELECT * FROM Users WHERE username = '$userName'";
+      $result = $this->conn->query($data);
+      if ($result && mysqli_num_rows($result) > 0) {
+        return $result->fetch_assoc();
+      }
+      return NULL;
+    }
+
+    /**
+     * This function is used to fetch the book list details of the reader from
+     * the database.
+     * 
+     *   @param string $userName
+     *     Stores the user name.
+     * 
+     *   @return mixed
+     *     Returns query result containing the names of the books of each list
+     *     if list exists otherwise NULL.
+     */
+    public function fetchReaderList(string $userName) {
+      $readingList = "SELECT * FROM reading_list WHERE username = '$userName' ORDER BY book_name ASC";
+      $bucketList = "SELECT * FROM bucket_list WHERE username = '$userName' ORDER BY book_name ASC";
+      $response = [];
+      $result1 = $this->conn->query($readingList);
+      $result2 = $this->conn->query($bucketList);
+      if ($result1 && mysqli_num_rows($result1) > 0) {
+        $response["readingList"] = $result1;
+      }
+      if ($result2 && mysqli_num_rows($result2) > 0) {
+        $response["bucketList"] = $result2;
+      }
+      if (count($response) > 0) {
+        return $response;
```

Comment:
no need to check for count > 0 , because if count is > 0 then it will be true. just use if (count)